src/ProjectTemplates/Web.ProjectTemplates/content/RazorComponentsWeb-CSharp/Shared/LoginDisplay.IndividualB2CAuth.razor

```diff
@@ -1,15 +1,3 @@
-@*#if (IndividualLocalAuth)
```

Comment:
@seancpeters do you know why this wouldn't work with `.razor`? Does this work with `.cshtml`?

Reply:
It looks like the templating engine does understand this https://github.com/dotnet/templating/blob/c758b031c16827a6d760f125705945c4cf5f9b9d/src/Microsoft.TemplateEngine.Orchestrator.RunnableProjects/SimpleConfigModel.cs#L414

Reply:
Yes, it works fine from the command line. It just doesn't work inside VS. I'm not sure why they would be different.

Reply:
The reason for this not working in VS would seem to be that the latest bits of dotnet/templating are not shipped with VS. I'm looking into this - and will try to get the bits updated in VS.